modules/widgets/twitter-timeline.php

```diff
@@ -84,59 +86,54 @@ public function admin_scripts( $hook ) {
 	 * @param array $instance Saved values from database.
 	 */
 	public function widget( $args, $instance ) {
+		$output = '';
+
 		// Twitter deprecated `data-widget-id` on 2018-05-25,
 		// with cease support deadline on 2018-07-27.
 		if ( isset( $instance['type'] ) && 'widget-id' === $instance['type'] ) {
 			if ( current_user_can( 'edit_theme_options' ) ) {
-				echo $args['before_widget'];
-				echo $args['before_title'] . esc_html__( 'Twitter Timeline', 'jetpack' ) . $args['after_title'];
-				echo '<p>' . esc_html__( "The Twitter Timeline widget can't display tweets based on searches or hashtags. To display a simple list of tweets instead, change the Widget ID to a Twitter username. Otherwise, delete this widget.", 'jetpack' ) . '</p>';
-				echo '<p>' . esc_html__( '(Only administrators will see this message.)', 'jetpack' ) . '</p>';
-				echo $args['after_widget'];
+				$output .= $args['before_widget']
+				. $args['before_title'] . esc_html__( 'Twitter Timeline', 'jetpack' ) . $args['after_title']
+				. '<p>' . esc_html__( "The Twitter Timeline widget can't display tweets based on searches or hashtags. To display a simple list of tweets instead, change the Widget ID to a Twitter username. Otherwise, delete this widget.", 'jetpack' ) . '</p>'
+				. '<p>' . esc_html__( '(Only administrators will see this message.)', 'jetpack' ) . '</p>'
+				. $args['after_widget'];
 			}
+
+			echo $output; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 			return;
 		}
 
 		$instance['lang'] = substr( strtoupper( get_locale() ), 0, 2 );
 
-		echo $args['before_widget'];
+		$output .= $args['before_widget'];
 
 		$title = isset( $instance['title'] ) ? $instance['title'] : '';
 
 		/** This filter is documented in core/src/wp-includes/default-widgets.php */
 		$title = apply_filters( 'widget_title', $title );
 		if ( ! empty( $title ) ) {
-			echo $args['before_title'] . $title . $args['after_title'];
-		}
-
-		if ( isset( $instance['type'] ) && 'widget-id' === $instance['type'] && current_user_can( 'edit_theme_options' ) ) {
```

Comment:
This conditional probably could never be `true`, as this [conditional above](https://github.com/Automattic/jetpack/pull/15328/files#diff-a3c30af2849ad5f1deb4b9c70f28bb2cL89):

```
if ( isset( $instance['type'] ) && 'widget-id' === $instance['type'] ) {
```
...would also be `true`, and it returns.